test/plots/piePlotTests.ts

```diff
@@ -449,7 +413,7 @@ describe("Plots", () => {
         svg.remove();
       });
 
-      it("points within innerRadius() and outside of outerRadius() don't return entities", () => {
+      it("returns nothing for points within innerRadius() or outside of outerRadius() with entitiesAt()", () => {
         piePlot.innerRadius(100).render();
         let click1 = { x: 200, y: 201 };
```

Comment:
If we could find a way to make sense of all these magic numbers it would be pretty cool, but I can't think of something.